Exit code: 1
Stdout:

Search operators to employ:
genetic_crossover(probability=0.8,mutation_rate=0.1,population_size=50) with greedy
genetic_mutation(probability=0.7,mutation_rate=0.2,population_size=40) with metropolis
--------------------------------------------------

Stderr:
Traceback (most recent call last):
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/ollama_output_20241003_091629/execution_iteration_6.py", line 35, in <module>
    met.run()
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/metaheuristic.py", line 143, in run
    self.apply_search_operator(perturbator, selector)
  File "/Users/valeriaenriquezlimon/Documents/research-llm/llm-metaheuristics/metaheuristic.py", line 99, in apply_search_operator
    exec('Operators.' + operator_name + '(self.pop,' + operator_params)
  File "<string>", line 1, in <module>
TypeError: genetic_crossover() got an unexpected keyword argument 'probability'
